utils/bash/ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I think I prefer putting this stuff in the rakefile just to keep tools separated from the repo.